RecoLocalMuon/CSCSegment/src/CSCCondSegFit.cc

```diff
@@ -240,7 +240,7 @@ void CSCCondSegFit::correctTheCovX(void){
     double chi2uCorrection = chiUZ/chi2Norm_;
     for(unsigned i=0; i<uu.size(); ++i)
       lex_[i]=lex_[i]*chi2uCorrection;
-      setScaleXError(chi2uCorrection);
```

Comment:
I am the person who added that specific line and I am pretty sure it 
should not be part of the loop. This addition was part of my replacement 
of CLHEP matrices by SMatrix. I think this 'set' is required in order to 
update the chi2 member value; the original code treated this variable as 
a global throughout the code. Note that the value 'set' doesn't depend 
on the loop. The indentation is certainly misleading!

Patrick Gartung wrote:
> In RecoLocalMuon/CSCSegment/src/CSCCondSegFit.cc
> <https://github.com/cms-sw/cmssw/pull/15007#discussion_r68773381>:
>
>>  @@ -240,7+240,7 @@ void CSCCondSegFit::correctTheCovX(void){
>>       double chi2uCorrection = chiUZ/chi2Norm_;
>>       for(unsigned i=0; i<uu.size();++i)
>>         lex_[i]=lex_[i]*chi2uCorrection;
>>  -      setScaleXError(chi2uCorrection);
>
> Is this line part of the for loop?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/cms-sw/cmssw/pull/15007/files/8239aafc617444a1ad014cacdf324ca9c40fda43#r68773381>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AE2Fnv-E3fWNYXTfA_LerCqy8MoSBZwqks5qQTYrgaJpZM4I_i87>.
>